app/README.md

```diff
@@ -0,0 +1,62 @@
+# Development Tips
+
+## Start the Rest API w/o security
+
+Steps:
+ 1. Start PostgreSQL from docker
+
+        $ docker run -d --rm -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password -e POSTGRES_DB=syndesis postgres
+
+ 2. Start Syndesis backend (I run io.syndesis.runtime.Application from IDE), with JVM options:
+ 
+         -Dendpoints.test_support.enabled=true -Ddeployment.load-demo-data=false -Dencrypt.key=hello
+
+ 3. Install tinyproxy
+        
+        $ brew install tinyproxy --with-reverse
+
+    Start [tinyproxy](https://tinyproxy.github.io/) from a directory that has `tinyproxy.conf` (below)
+
+        $ tinyproxy -d -c tinyproxy.conf
+
+ 4. Modify the `$SYNDESIS/app/ui/src/config.json` (sample config is in the [repo](https://github.com/syndesisio/syndesis/blob/master/app/ui/src/config.json.example))
+    the `apiEndpoint` needs to point to tinyproxy (`http://localhost:9090`):
+    
+        "apiBase": "http://localhost:9090"
+     
+     or
+
+        "apiEndpoint": "http://localhost:9090/api/v1",
+        "mapperEndpoint": "http://localhost:9090/mapper/v1",
+
+  5. Start the UI with from `$SYNDESIS/app/ui` (You may need to `npm rebuild node-sass`)
```

Comment:
If this is strictly related to the UI should this information maybe be added to `app/ui/README.md` instead?

Reply:
@gashcrumb seems easier to me to keep it all in one place.

Reply:
Shouldn't that be the 'developer handbook' then?  Not entirely sure where that gets it's content from...